x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -792,6 +796,10 @@ export class TaskRunner<
     eventLogger.logEvent(event);
 
     if (!this.cancelled) {
+      this.inMemoryMetrics.increment(IN_MEMORY_METRICS.RULE_EXECUTIONS);
```

Comment:
Do we want to keep track of timeouts as well?

Reply:
Yea, that'd be a great addition